src/Utils/negative_tracers.jl

```diff
@@ -60,14 +62,17 @@ github](https://github.com/OceanBioME/OceanBioME.jl/discussions/48).
 
 Future plans include implement a positivity-preserving timestepping scheme as the ideal alternative.
 
-If `warn` is true then scaling will raise a warning.
+~~If `warn` is true then scaling will raise a warning.~~
+
+`invalid_fill_value` specifies the value to set the total cell content to if the total is less than 0
+(meaning that total tracer conservation can not be enforced).
```

Comment:
perhaps we elaborate a bit more here... eg mention the caveat you included in the first comment in the PR? also, if other models follow similar practice perhaps mention 1-2 here?